usaspending_api/awards/v2/views/funding_rollup.py

```diff
@@ -20,7 +20,9 @@
                 faba.treasury_account_id
         from    awards a
                 inner join financial_accounts_by_awards faba on faba.award_id = a.id
-        where   {award_id_column} = {award_id}
+                INNER JOIN submission_attributes sa ON faba.submission_id = sa.submission_id
+                INNER JOIN dabs_submission_window_schedule dabs ON sa.submission_window_id = dabs.id
+        where   {award_id_column} = {award_id} and dabs.submission_reveal_date <= now()
```

Comment:
The other places changed in #2793 using this new FK places `submission_reveal_date <= now()` into the join clause. Did you discover performance issues doing that here?

Reply:
I did not. I will test performance of that strategy in order to maintain consistency.